examples/load_contacts_and_show_some_layers.py

```diff
@@ -51,16 +57,11 @@ def show_layers(popdict,show_ages=False):
     # load population into a dictionary of individuals who know who their contacts are
     options_args = {'use_microstructure': True}
     network_distr_args = {'Npop': n}
-    contacts = sp.make_contacts(location=location,state_location=state_location,country_location=country_location,options_args=options_args,network_distr_args=network_distr_args)
+    contacts = sp.make_contacts(location=location, state_location=state_location, country_location=country_location,
+                                options_args=options_args, network_distr_args=network_distr_args)
 
     # not all school and workplace contacts are going to be close contacts so create 'closer' contacts for these settings
     close_contacts_number = {'S': 20, 'W': 20}
-    contacts = sp.trim_contacts(contacts,trimmed_size_dic=close_contacts_number)
-
-
-    verbose = True
-    # verbose = False
-    if verbose:
-        show_layers(contacts,show_ages=True)
-
+    contacts = sp.trim_contacts(contacts, trimmed_size_dic=close_contacts_number)
 
+    show_layers(contacts, show_ages=False)
```

Comment:
change to sp.show_layers(contacts, show_ages=False), maybe add line sp.show_layers(contacts, show_ages=True)